queue_services/entity-filer/src/entity_filer/filing_processors/dissolution.py

```diff
@@ -95,6 +97,15 @@ def process(business: Business, filing: Dict, filing_rec: Filing, filing_meta: F
             'dissolutionDate': LegislationDatetime.format_as_legislation_date(business.dissolution_date)
         }
 
+    # update batch processing entry, if any is present
+    if flag_on:
+        batch_processings = BatchProcessing.find_by(business_id=business.id)
```

Comment:
i think you should search the batch processing entry by filing id.

Reply:
Updated and confirmed it works locally + unit tests still pass locally